### DISCUSSION TITLE
Is this project alive?

### DISCUSSION THREAD

**Discussion by AdiAbuAli** [mrlt8/docker-wyze-bridge#288] [General]
Am wondering if this project reached End of maintenance phase ?
Its nice one and sad to see it with no updates.

**Comment by mrlt8** [comment]
I've been busy on another project, but I should have a little more free time to push some updates for this one.

Does anyone have any info on the new forced firmware updates coming soon?

**Reply by tcnasc** [reply]
No, I don’t know about any forced fw updates

**Comment by AdiAbuAli** [comment]
I have no idea about this forced FW

Thanks,
________________________________

**Comment by mrlt8** [comment]
hmm. I think some of my cams on the WyzeHacks firmware auto-updated to the DTLS firmware...

**Comment by AdiAbuAli** [comment]
It working fine for me the edge..
When datanot ready it restart...

Anyways, (different subject), noticed that streaming is not smooth compare to before DTLS FW (alot of frame not available)....anyways, at least from connection perspective its much better

Thanks,
________________________________

**Comment by AdiAbuAli** [comment]
Also wondering what is WebRTC option will be used for?

Thanks,
________________________________

**Reply by mrlt8** [reply]
For now, it just spits out the credentials to connect to the WebRTC stream.
